coffea/processor/servicex/accumulator.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2019, IRIS-HEP
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+import copy
+from typing import Dict, Any
+
+from coffea.processor.accumulator import dict_accumulator, defaultdict_accumulator
+
+
+class Accumulator(dict_accumulator):
+    """
+    Simplified interface to Coffea accumulator. Sets up the most basic form automatically
+    """
+
+    def __init__(self, cats: Dict[str, Any]):
```

Comment:
Migrate this to the new accumulator semantics.
https://github.com/CoffeaTeam/coffea/blob/master/tests/test_processor.py#L94-L111

Reply:
I think once that is done this whole file can go away

Reply:
I'll delete this for now - I'd like to have a conversation about the trade-off of the flexibility of using a generic python interface (Accumulatable) vs the usability of a specific base class. Returning a dictionary of histograms is a bit harder to learn than returning an instance of a class with clear constructors.

Reply:
With the new style it is also possible to return a single hist, and in general any nested dictionary of objects implementing add. This makes coffea maintenance easier because we don't have to create a wrapper/subclass/specialization/etc. for outside things like `hist` package objects. If it can be added, we'll add it up!